src/components/modules/CodeRequest/modules/CodeView/modules/SmsCode/SmsCode.view.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <form
+    data-test="sms-code-form"
+    @submit.prevent="onSubmit"
+  >
+    <v-title>
+      {{ $t('components.smsCode.title') }}
+    </v-title>
+    <v-description>
+      {{ $t('components.smsCode.description') }}
+    </v-description>
+
+    <form-item>
+      <v-input
+        v-model="code"
+        v-validate="'required|digits:6'"
+        data-vv-as="code"
+        data-vv-name="code"
+        :error="errors.first('code')"
+        name="code"
+        :placeholder="$t('components.smsCode.enterReceivedCode')"
+        data-test="code-input"
+      />
+    </form-item>
+    <form-item class="v-mb-24">
+      <v-button
+        :disabled="!isFormValid || isLoading"
+        :is-loading="isLoading"
+        type="submit"
+        data-test="submit-button"
+      >
+        {{ $t('global.confirm') }}
+      </v-button>
+    </form-item>
+    <form-row class="v-fs-14 v-text-center">
+      <send-code
+        :is-loading="isLoading"
+        @click="sendCode"
+      />
+    </form-row>
+  </form>
+</template>
+
+<script>
+import VButton from '@endpass/ui/kit/VButton';
+import VInput from '@endpass/ui/kit/VInput';
+import SendCode from '@/components/common/SendCode';
+import FormItem from '@/components/common/FormItem';
+import FormRow from '@/components/common/FormRow';
+import formMixin from '@/mixins/form';
+import VTitle from '@/components/common/VTitle';
+import VDescription from '@/components/common/VDescription';
+
+export default {
+  name: 'SmsCodeView',
+
+  props: {
+    isLoading: {
+      type: Boolean,
+      default: false,
+    },
+
+    error: {
+      type: String,
+      default: '',
+    },
+  },
+
+  data: () => ({
+    code: '',
+  }),
+
+  watch: {
+    error: {
+      handler(msg) {
+        this.$validator.errors.removeById('sendCodeId');
```

Comment:
Have written about that.

Reply:
have some problems with this
https://github.com/endpass/endpass-auth/pull/278#discussion_r380460908